specification/subscription/resource-manager/readme.md

```diff
@@ -146,9 +146,23 @@ output-folder: $(azure-libraries-for-java-folder)/azure-mgmt-subscription
 
 ``` yaml $(java) && $(multiapi)
 batch:
+  - tag: package-2019-10-preview
```

Comment:
just curious what's the difference between this subs and the subs inside resources?

Reply:
I don't know. I think it depends on services team. But the subs inside resources have no version 2019-10-preview.